### ISSUE TITLE
Should this repo be deleted?

### ISSUE THREAD

**Issue by jeffwidman** [dependabot/vgotest#1]
I'm trying to debug a problem with go modules, and it's confusing seeing this repo... is it actively used (or planned to be used) or should it be deleted?

**Comment by jeffwidman** [comment]
nudge @jurre

**Comment by jurre** [comment]
I'm not sure if it's actively used, but we do have a tonne of references to it in our dependabot-core test fixtures and I'm worried that deleting the repo will cause those tests to fail.

**Comment by jeffwidman** [comment]
Ah yeah, I didn't even think to check for those since I assumed this repo was some standalone experimentation thing. Makes sense. If you don't mind, maybe hold this ticket open for a bit as a reminder to see if those fixtures are easy to eventually update to drop this dep.

**Comment by jeffwidman** [comment]
Note to self: See also https://github.com/dependabot/dependabot-core/commit/36daf87248712ba69febfb289d12b73b12d30583

That said, the entire directory https://github.com/dependabot/dependabot-core/commits/main/go_modules/spec/fixtures/go_mods looks like it predates storing all the test repos in https://github.com/dependabot/dependabot-core/tree/main/go_modules/spec/fixtures/projects

So I have a funny feeling this can be migrated to https://github.com/dependabot/dependabot-core/tree/main/go_modules/spec/fixtures/projects and then removed... 

If possible, that would certainly reduce confusion...

**Comment by jurre** [comment]
> looks like it predates storing all the test repos in https://github.com/dependabot/dependabot-core/tree/main/go_modules/spec/fixtures/projects

Yep it does, there are a few tests lingering around that still reference those old files, but I agree that migrating everything to the project based fixtures would be good!
